testsuite/OMSimulator/rename.lua

```diff
@@ -0,0 +1,147 @@
+-- status: correct
+-- linux: yes
+-- mingw: yes
+-- win: no
+-- mac: no
+
+oms_setCommandLineOption("--suppressPath=true")
+oms_setTempDirectory("./multipleinputconnections/")
+
+oms_newModel("model")
+
+oms_addSystem("model.root", oms_system_wc)
+
+oms_addSystem("model.root.System1", oms_system_sc)
+
+oms_addSubModel("model.root.add", "../resources/Modelica.Blocks.Math.Add.fmu")
+
+src = oms_list("model")
+print(src)
+
+status = oms_rename("model.root", "root_1")
+status = oms_rename("model.root.System1", "System_1")
+status = oms_rename("model.root.add", "add_1")
+
```

Comment:
I am a bit confused. Shouldn't it be `oms_rename("model.root_1.System1", "System_1")` and `oms_rename("model.root_1.add", "add_1")`?

Reply:
@lochel this should be fixed with this https://github.com/OpenModelica/OMSimulator/pull/928/commits/ad744631b2fc5b3e8cb13a324e98d5c0b0e7ccb5